### PR TITLE
did:web documentation

### PR DESCRIPTION
**Pull request by jonmattgray**
https://digicatapult.atlassian.net/browse/VR-29

Instructions for a user to generate and setup the repo to use their own web DIDs.
Script that imports DIDs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,119 @@ With the default configurations defined in this repo's `.env.*` files, the porta
...
+```
+
+Use the `x` value for `"x": "PUBLIC_KEY"` in the DID. Save the private `d` value to be used later.
+
+GitHub Pages can be used to host web DIDs for free:
+
+1.  Create a public repo called `YOUR_USERNAME.github.io`
+2.  In the repo, create your desired directory structure e.g. `dids/test/alice`
+3.  Add a `did.json` in the nested directory.
+4.  After pushing to GitHub, the DID can be viewed e.g. https://jonmattgray.github.io/dids/test/alice/did.json. Note the directory structure matches the did `did:web:jonmattgray.github.io:dids:test:alice`.
```
Comment (Ellenn-A) [inline README.md:129]: here again probs don't include your actual username

File: README.md [hunk 2]
```diff
@@ -55,7 +55,119 @@ With the default configurations defined in this repo's `.env.*` files, the porta
...
+
+The DID and PRIVATE_KEY of each persona is then set using env files at the root of this repo. For example:
+
+`.env.nice-agent-alice.local`
+
+```
+DID=did:web:jonmattgray.github.io:dids:test:alice
```
Comment (Ellenn-A) [inline README.md:136]: ...same as above

File: README.md [hunk 3]
```diff
@@ -55,7 +55,119 @@ With the default configurations defined in this repo's `.env.*` files, the porta
...
+`.env.nice-agent-issuer.local`
+
+```
+DID=did:web:jonmattgray.github.io:dids:test:charlie
+PRIVATE_KEY=PRIVATE_KEY
+```
+
+#### Importing DIDs
+
+Finally, before forming connections, each DID needs to be imported to its cloudagent using the `dids/import` endpoint. Run the following script, which automatically uses DIDs and private keys from the `.local` env files to import all three personas:
```
Comment (Ellenn-A) [inline README.md:156]: I know this is over-explaining but we could mention that one should have all the containers up before trying to run the script

File: README.md [hunk 4]
```diff
@@ -55,7 +55,119 @@ With the default configurations defined in this repo's `.env.*` files, the porta
...
+
+An example DID (Alice):
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
+  ],
+  "id": "did:web:jonmattgray.github.io:dids:test:alice",
```
[the remark this answers is not in the document]
Comment (jonmattgray) [reply README.md]: all changed to `YOUR_USERNAME`

File: README.md (hunk 3 above)
Comment (jonmattgray) [reply README.md]: added `With containers up using 'npm run compose:up:test'...`

**Review by Ellenn-A — APPROVED** [review]
looks good to me

**Pull request merged**